### PR TITLE
Opeted-In cookie global

### PR DESCRIPTION
**Pull request by chessky**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java [hunk 1]
```diff
@@ -276,15 +268,16 @@ public static void start(Method testMethod) {
...
   private void stopLogging() {
     driver = DriverProvider.getActiveDriver();
-    if ( driver.getProxy() != null && Configuration.getForceHttps()) {
+    if (driver.getProxy() != null && Configuration.getForceHttps()) {
       Har har = driver.getProxy().getHar();
       for (HarEntry entry : har.getLog().getEntries()) {
         URL url = null;
         try {
           url = new URL(entry.getRequest().getUrl());
           if (url.getHost().contains("wikia")) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java]: MINOR Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS134)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L159](https://github.com/Wikia/selenium-tests/blob/9033c0abbdee1f6c08d6404c287fb2d0816178c1/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L159): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java [hunk 2]
```diff
@@ -87,7 +71,10 @@ public static void log(String command, String description, boolean success, WebD
     new Shooter().savePageScreenshot(screenPath + imageCounter, driver);
     CommonUtils.appendTextToFile(screenPath + imageCounter + ".html", getPageSource(driver));
     String className = success ? SUCCESS_CLASS : ERROR_CLASS;
-    String html = VelocityWrapper.fillLogRowWithScreenshot(Arrays.asList(className), command, description, imageCounter);
+    String
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java]: Looks like you have broken style guides on your computer. Please follow steps to install: https://github.com/Wikia/guidelines/tree/master/Java

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L159](https://github.com/Wikia/selenium-tests/blob/296e22aaa68e9be174c256aac11813025358e792/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L159): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**